### PR TITLE
edit and delete release plan

### PR DESCRIPTION
**Pull request by ngaa1leloi**
edit and delete release plan

### CODE REVIEW

File: app/Http/Controllers/User/ReleaseController.php [hunk 1]
```diff
@@ -86,7 +86,9 @@ public function store(ReleaseRequestStore $request)
      */
     public function show($id)
     {
-        //
+        /*$release = $this->release->find($id);
+
+        return view('release.show', compact('release'));*/
```
Comment (nguyenngocsond) [inline app/Http/Controllers/User/ReleaseController.php]: không dùng thì xoá đi

File: resources/views/release/edit.blade.php [hunk 2]
```diff
@@ -0,0 +1,95 @@
+@extends('layouts.app')
+
+@section('content')
+<div class="m-grid__item m-grid__item--fluid m-wrapper">
+    <!-- BEGIN: Subheader -->
+    <div class="m-subheader">
+        <div class="d-flex align-items-center">
+            <div class="mr-auto">
+                <h3 class="m-subheader__title ">
+                    Create New Release Plan
```
Comment (nguyenngocsond) [inline resources/views/release/edit.blade.php]: I18n

File: resources/views/release/edit.blade.php [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+                                <span class="input-group-addon">
+                                    <i class="la la-calendar-check-o"></i>
+                                </span>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="form-group m-form__group">
+                        <label>
+                            <h5>
+                                Goal
```
Comment (nguyenngocsond) [inline resources/views/release/edit.blade.php]: i18n

File: app/Http/Controllers/User/ReleaseController.php [hunk 4]
```diff
@@ -129,6 +137,10 @@ public function update(Request $request, $id)
      */
     public function destroy($id)
     {
-        //
+        $release = $this->release->delete($id);
+
+        Toastr::success('Release Successfully Deleted', 'Success');
```
Comment (BunnyPi04) [inline app/Http/Controllers/User/ReleaseController.php]: i18n

File: resources/views/release/index.blade.php [hunk 5]
```diff
@@ -55,13 +55,13 @@
                                         </span>
                                     </a>
                                 </li>
-                                <li class="m-portlet__nav-item">
-                                    <a href="{{ route('user.release.destroy', $release->id) }}" class="btn m-btn--pill btn-danger" data-toggle="tooltip" data-placement="bottom" title="Delete">
+                                {{ Form::open([ 'method' => 'DELETE', 'route' => [ 'user.release.destroy', $release->id], 'class' => 'm-portlet__nav-item']) }}
+                                    <button type="submit" class="btn m-btn--pill btn-danger" data-toggle="tooltip" data-placement="bottom" title="Delete">
```
Comment (BunnyPi04) [inline resources/views/release/index.blade.php]: Laravel collective

**Pull request merged**